addons/l10n_in/models/mail_message.py

```diff
@@ -24,8 +26,8 @@ def _compute_l10n_in_audit_log_preview(self):
                 title = message.subtype_id.display_name
             audit_log_preview = Markup("<div>%s</div>") % (title)
             for value in tracking_value_ids:
-                audit_log_preview += Markup("<li>%s <i class='o_TrackingValue_separator fa fa-long-arrow-rightmx-1 text-600' title='%s' role='img' aria-label='%s'></i>%s (%s)</li>") % (
-                    _("Changed"), _("Changed"), value._get_old_display_value()[0], value._get_new_display_value()[0], value.field.field_description)
+                audit_log_preview += Markup("<li>%s <i class='o_TrackingValue_separator fa fa-long-arrow-right mx-1 text-600' title='%s' role='img' aria-label='%s'></i>%s (%s)</li>") % (
+                    value._get_old_display_value()[0] or none, arrow_title, arrow_title, value._get_new_display_value()[0] or none, value.field.field_description)
```

Comment:
https://github.com/odoo/odoo/pull/121242#discussion_r1201965066 hasn't been addressed?
```suggestion
                audit_log_preview += Markup(
                    "<li>%(value)s <i class='o_TrackingValue_separator fa fa-long-arrow-right mx-1 text-600' title='%(title)s' role='img' aria-label='%(title)s'></i>%(value)s (%(field)s)</li>"
                ) % {
                    'value': value._get_old_display_value()[0] or _("None"), 
                    'title': _("Changed"),
                    'field': value.field.field_description,
                }
```

There is also no need to declare the variable higher (except for minor speed improvement) because nothing is repeated.